downstream/modules/aap-on-azure/proc-peer-vwan-hub-to-aap.adoc

```diff
@@ -0,0 +1,33 @@
+[id="proc-peer-vwan-hub-to-aap"]
+
+= Peering a VWAN Hub to the {AAPonAzureNameShort} Network
+
+Before you can peer a VWAN with the {AAPonAzureNameShort} VNET, you must have connected a hub network and at least one spoke network to the hub network of the Azure VWAN that you intend to connect {AAPonAzureNameShort} to.
+
+.Prerequisites
+
+* A pre-configured Azure VWAN that is already tested and working.
+* One or more of the following connections to the VWAN:
+** A DMZ network that contains Azure virtual machines that users will use to access {AAPonAzureNameShort}.
+** A DMZ network that contains an Azure virtual machine that can be used with SSH tunneling to access {AAPonAzureNameShort}.
+** A VPN or Direct Connect service to your local network that routes traffic from machines on your local network to {AAPonAzureNameShort}.
+
+Peering the {AAPonAzureNameShort} VNET to your VWAN follows the same approach as peering any other VNET or network with a VWAN.
+
+. Navigate to the *Virtual Network Connections* page for the VWAN that you want to peer with your {PlatformNameShort} instance.
+. Use the following settings to create a connection between the VWAN hub and your {PlatformNameShort} instance:
```

Comment:
```suggestion
. Create a connection between the VWAN hub and your {PlatformNameShort} instance with the following settings:
```

Reply:
I often try to put the thing that the user is doing before the tools that they use to do it.

In this case, the user wants to create a connection not use the settings. Know what I mean?